src/collections/_documentation/platforms/cocoa/init/objc.md

```diff
@@ -2,9 +2,9 @@
 @import Sentry;
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    [SentrySDK startWithOptions:@{
-        @"dsn": @"___PUBLIC_DSN___",
-        @"debug": @(YES)
+    [SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
```

Comment:
if they are on the older version, this will likely not work.

What's about doing something like:

Version <= do like this:
...
Version >= do like this:
...

Reply:
Very good point 👏